app/components/edit/FormTextArea.jsx

```diff
@@ -4,14 +4,14 @@ import PropTypes from 'prop-types';
 const FormTextArea = ({
   label, placeholder, value, setValue,
 }) => (
-  <li className="edit--section--list--item">
+  <React.Fragment>
```

Comment:
woo! 16!

Reply:
I kind of prefer (but totally up to you)

`import React, { Fragment } from 'react';`

```
<Fragment>
...
</Fragment>
```

Reply:
I have no strong preference, so I went ahead and changed it to `<Fragment>`.

I am kind of interested in the `<></>` syntax for fragments, but it looks like that'll require a Babel upgrade.